deploy/role_binding.yaml

```diff
@@ -1,14 +1,14 @@
-kind: RoleBinding
+kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: metal3-baremetal-operator
-  namespace: metal3
 subjects:
 - kind: ServiceAccount
   name: metal3-baremetal-operator
+  namespace: metal3
 - kind: User
   name: developer
 roleRef:
-  kind: Role
+  kind: ClusterRole
   name: metal3-baremetal-operator
```

Comment:
This is still defined as a Role, not a clusterRole

Reply:
fixed